x-pack/plugins/security_solution/common/endpoint/models/policy_config.ts

```diff
@@ -241,6 +241,10 @@ export const policyFactoryWithoutPaidFeatures = (
         reputation_service: false,
         supported: false,
       },
+      antivirus_registration: {
+        mode: AntivirusRegistrationModes.disabled,
+        enabled: false,
+      },
```

Comment:
aren't we want to keep this enabled for unpaid users? malware protection is still set to prevent as far as i understand

Reply:
Good call, I believe you are right. Just to double chceck - @caitlinbetz could you confirm that for unpaid users we still want to display this setting by default (enabled, sync with malware).

Reply:
Confirmed with @caitlinbetz offline that we want the default of enabled, sync with malware for both free and paid tiers.

Reply:
awesome, thank you for proactively looking into it! 🙌